net.tenshu.Terminator2.yml

```diff
@@ -0,0 +1,47 @@
+---
+id: net.tenshu.Terminator2
+base-version: main
+runtime: org.gnome.Sdk
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: terminator
+rename-desktop-file: terminator.desktop
+rename-appdata-file: terminator.appdata.xml
+rename-icon: terminator
+copy-icon: true
+
+finish-args:
+  - --share=ipc
+  - --socket=session-bus
```

Comment:
```suggestion
```

This is not allowed.

Reply:
@hfiguiere is there an alternative I could use? Application doesnt work without it.